DependencyInjection/Configuration.php

```diff
@@ -111,7 +111,17 @@ private function getMailersNode()
                     ->end()
                 ->end()
                 ->scalarNode('sender_address')->end()
-                ->scalarNode('delivery_address')->end()
+                ->variableNode('delivery_address')
```

Comment:
this should not be a variable node. It should be an prototyped array of scalar nodes (named ``delivery_addresses``), with the XML plural support and a normalizer for the old config (which may not be necessary because of the XML plural support which may already catch it btw)